peer/chaincode.proto

```diff
@@ -85,3 +87,44 @@ message ChaincodeInvocationSpec {
 message LifecycleEvent {
     string chaincode_name = 1;
 }
+
+// https://github.com/hyperledger/fabric/blob/master/core/common/ccprovider/cdspackage.go#L36
+//CDSData is data stored in the LSCC on instantiation of a CC
```

Comment:
Please use [slash][slash][space] for the doc comments. I understand that we don't do that in all comments but we're working towards that.

Reply:
done